src/utils.jl

```diff
@@ -548,7 +548,7 @@ function dof_to_node(dh::Ferrite.AbstractDofHandler, u::Vector{T}; field_name=:u
         dof_range_ = Ferrite.dof_range(fh, field_name)
         for cell in Ferrite.CellIterator(dh,fh.cellset)
             _celldofs = Ferrite.celldofs(cell)
-            local_celldofs_field = reshape(@view(_celldofs[dof_range_]), (field_dim,length(cell.nodes)))
+            local_celldofs_field = reshape(@view(_celldofs[dof_range_]), (field_dim, length(dof_range_)÷field_dim))
```

Comment:
What exactly is fixed here? Both statements should be equivalent.

Reply:
Ah, bad title - makes it work for superparameteric elements (and subparametric).

Reply:
But see now that this would fail for subparametric and relies on that the linear dofs comes first (which they do). So probably a more complete solution would be desired...?